modules/opencensus/src/test/java/org/apache/ignite/internal/processors/monitoring/opencensus/OpenCensusDiscoveryTracingTest.java

```diff
@@ -286,8 +290,25 @@ public void testClientNodeStopTracing() throws Exception {
      */
     private void performClientBasedOperation(
         @Nullable Runnable preparationOperations,
-        @NotNull Runnable operationToCheck) throws Exception
-    {
+        @NotNull Runnable operationToCheck
+    ) throws Exception {
+        performClientBasedOperation(preparationOperations, operationToCheck, 0);
+    }
+
+    /**
+     * Perform preparation operations and operation to trigger span creation.
+     *
+     * @param preparationOperations Operations to run without tracing.
+     * @param operationToCheck Operations to run with tracing.
+     * @param nodeIdx Node id that that is used for changing tracing configuration.
```

Comment:
If you'll continue with same-node-solution please add more meaningful comment that will describe why it's needed.

Reply:
Ok, will do.